### PR TITLE
add Idris author markdown

### PR DESCRIPTION
**Pull request by in-odds**
### Description

Adds my personal profile details
 
### Issues Resolved

None

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/osci_2023/in-odds.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Idris Yalaletdinov
+short_name:
```
Comment (joshuarrrr) [inline _authors/osci_2023/in-odds.markdown:3]: remove if not needed:
```suggestion
```

**Review by joshuarrrr — CHANGES_REQUESTED** [review]
Remove the `short_name` field. Otherwise looks good.

**Comment by dtaivpp** [comment]
Hey @in-odds please review Josh's feedback, thank you!

File: _authors/osci_2023/in-odds.markdown (hunk 1 above)
Comment (in-odds) [reply _authors/osci_2023/in-odds.markdown]: Hello, okay, thanks

**Comment by in-odds** [comment]
Hello @dtaivpp I removed short name, can you check it

**Review by joshuarrrr — APPROVED** [review]
LGTM

**Review by dtaivpp — APPROVED** [review]
Looks good!

**Pull request merged**